src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -122,7 +123,8 @@ class ActiveFocusInvestigation extends React.Component<
 
     const { caseTriggeredPlans, routinePlans } = this.props;
     if (
-      (caseTriggeredPlans && caseTriggeredPlans.length === 0) ||
+      caseTriggeredPlans &&
+      caseTriggeredPlans.length === 0 &&
```

Comment:
Am not sure what is happening here, 
might need some extra :eyes:  cause i feel logically it makes sense to render plans when __either__ casetriggeredPlans > 0 or routinePlans > 0. I think this change would render the plans if and only if we have atleast one of each of caseTriggeredPlans and routinePlans.